frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -672,6 +673,17 @@ export function getAnnotationInformation(
   return Request.receiveJSON(infoUrl, options);
 }
 
+export function getSandboxAnnotationInformation(
```

Comment:
> What exactly is the semantics of this? I'm a bit confused why it says AnnotationInformation, but its signature only seems to be specific to a dataset. Does it create an empty dummy annotation for that dataset?

This corresponds to the `getAnnotationInformation` function directly above it. Instead of fetching an existing specific annotation with an id, wK fetches an empty sandbox annotation (with all permissions set to false) for the specified dataset.

> How about getEmptySandboxAnnotationInformation then? Because the "classic sandbox mode" can also be activated for a non empty annotation, as far as I know..

The "classic sandbox mode" is no longer supported (at least not via the get parameter), but you're right, it's still possible to get into the sandbox mode by using the "Disable Saving" functionality. However, that is reset after reloading the page. I'd be fine with renaming it to `getEmptySandboxAnnotationInformation`.